hw02-generic/src/main/java/homework/CustomerReverseOrder.java

```diff
@@ -0,0 +1,16 @@
+package homework;
+
+import java.util.LinkedList;
+
+public class CustomerReverseOrder {
+
+    LinkedList<Customer> reverseCustomer = new LinkedList<Customer>();
```

Comment:
`private` и лучше объявлять тип минимально необходимым интерфейсом:
```suggestion
    private Deque<Customer> reverseCustomer = new LinkedList<Customer>();
```